webpack/scenes/ContentViews/components/IconWithCount.scss

```diff
@@ -0,0 +1,3 @@
+.ktable-cell-icon {
+  margin: 0 8px 0 0;
```

Comment:
I can never remember the order for these CSS shortcuts (top bottom left right? top right bottom left?)  When it's just a single value, would you be open to just using `margin-left` or whatever it is?